GoogleTestAdapter/VsPackage.Shared/OptionsPages/GeneralOptionsDialogPage.cs

```diff
@@ -14,9 +13,9 @@ public partial class GeneralOptionsDialogPage : NotifyingDialogPage
     {
         #region Test execution
 
-        [Category(SettingsWrapper.CategoryTestExecutionName)]
-        [DisplayName(SettingsWrapper.OptionTestDiscoveryRegex)]
-        [Description(SettingsWrapper.OptionTestDiscoveryRegexDescription)]
+        [LocalizedCategory("CategoryTestExecutionName")]
```

Comment:
It appears to me that this change is not strictly necessary, as the code as before would have been reading localized strings from SettingsWrapper. What is the benefit of this indirection?

Reply:
Attributes parameters must be compile-time constant, but the localized strings are not known until runtime. The indirection allows us to defer actual localized string lookup until runtime, while still having a constant parameter in the attribute.

Reply:
Makes sense, thanks!